linker/linker_test.go

```diff
@@ -1403,7 +1404,7 @@ func removePrefixIndent(s string) string {
 	}
 	lines = lines[1:] // skip first blank line
 	// determine whitespace prefix from first line (e.g. five tabstops)
-	var prefix []rune
+	prefix := make([]rune, 0, utf8.RuneCountInString(lines[1]))
```

Comment:
This seems wrong. This will always be a waste of space and much larger than is needed. We're only getting the whitespace prefix here, not the whole line.

Making this correct seems like a net reduction in code readability for basically zero benefit given that this is test code and this loop is only run like 100 times.